src/lightning/pytorch/strategies/fsdp.py

```diff
@@ -203,12 +202,9 @@ def num_processes(self) -> int:
 
     @property
     def process_group(self) -> Optional[ProcessGroup]:
```

Comment:
I would suggest removing this property, since we don't use it and it'll be a problem if we ever finish the collectives refactor

Reply:
It's easy to avoid the breaking change here the way I did, and so I would like to avoid it unless absolutely necessary. If we need to remove it in the future, we can always do it with the justification needed at that time.

Reply:
If you still think it should be removed, could you make the PR for it?

Reply:
Done: https://github.com/Lightning-AI/lightning/pull/18592